docker-compose.yml

```
@@ -18,6 +18,8 @@ services:
         - TZ=${TZ}
     volumes:
       - ./nginx/templates:/etc/nginx/templates
+      - ./nginx/server/certs/:/var/www/certs/
+      - ./docker/conf/customs/:/etc/nginx/conf.d/
```

```suggestion
      - ./nginx/server/certs/:/var/www/certs
      - ./nginx/conf/customs/:/etc/nginx/conf.d/customs
```

@pxthinh Please update the logic to change the path